src/main/java/com/redhat/cloud/notifications/models/CurrentStatus.java

```diff
@@ -0,0 +1,71 @@
+package com.redhat.cloud.notifications.models;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.redhat.cloud.notifications.db.converters.StatusConverter;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+import static com.fasterxml.jackson.annotation.JsonFormat.Shape.STRING;
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+import static com.fasterxml.jackson.databind.PropertyNamingStrategies.SnakeCaseStrategy;
+
+@Entity
+@Table(name = "status")
+@JsonNaming(SnakeCaseStrategy.class)
+public class CurrentStatus {
+
+    /*
+     * We don't need this field in the Java code but Hibernate needs each @Entity to have a field annotated with @Id.
+     * This field has a real purpose in the database though: it is used to guarantee that the status table will never
+     * contain more than one row. This is done with a combination of PK and CHECK SQL constraints.
+     */
+    @Id
+    @JsonIgnore
+    private boolean preventMultipleRows = true;
+
+    @NotNull
+    @Convert(converter = StatusConverter.class)
+    @Column(name = "value")
+    public Status status;
```

Comment:
Should we validate (somewhere) that when `status = MAINTENANCE` is set, startTime and endTime are required?